.envrc

```diff
@@ -0,0 +1,5 @@
+if ! has nix_direnv_version || ! nix_direnv_version 3.0.6; then
```

Comment:
i don't like the hardcoded versions here?
is there a better way?
or can we just land flake.nix ?

Reply:
To explain what's going on, direnv naturally supports the `use flake` directive. However, with the default implementation, we are doomed to re-computing the flake each time we get in/out of the directory.

This line allows to use an implementation that is made by the nix community and which caches the flake's build.

As for the version check, I guess we could remove it. I'm not very familiar with the previous versions, so I don't really know what this will imply.